### DISCUSSION TITLE
What happened to the Bookmark button?

### DISCUSSION THREAD

**Discussion by peterhoang** [danny-avila/LibreChat#3765] [Q&A] [answered]
I just updated to the latest [ea5140f], and I noticed the bookmark button is gone.

**Comment by danny-avila** [comment]
![image](https://github.com/user-attachments/assets/a73ea1d9-de22-43e4-bdfe-5e07fb5d3caa)
?

**Comment by danny-avila** [comment]
Ah I see the issue, I added RBAC but it's not defaulting to true if undefined for non-admins

**Reply by danny-avila** [reply, marked as answer]
For now you can enable it in librechat.yaml:

```yaml
interface:
  bookmarks: true
  prompts: true
```
